docs/usage.md

```diff
@@ -83,29 +226,293 @@ nextflow pull nf-core/oncoanalyser
+| ----------------------- | ------------------------- | --------------------------------------- |
+| Genomes and indexes     | `conf/hmf_genomes.config` | Absolute paths                          |
+| hmftools resource files | `conf/hmf_data.config`    | Paths relative to data bundle directory |
+| Panel resource files    | `conf/panel_data.config`  | Paths relative to data bundle directory |
+
+See the below sections for further details on customising reference data.
+
+#### Customising hmf data
+
+To override hmftools resource files, first [stage the bundle](#staging-reference-data) locally then copy in your
+custom file under the bundle directory and create a new config with relevant file paths:
+
+```text title="hmf_data.custom.config"
+params {
+    hmf_data_paths {
+        '38' {
+            driver_gene_panel     = 'custom_files/DriverGenePanel.tsv'
+            sage_actionable_panel = 'custom_files/ActionableCodingPanel.bed.gz'
+            sage_coverage_panel   = 'custom_files/CoverageCodingPanel.bed.gz'
+        }
+    }
+}
+```
+
+To use these hmftools resource file overrides in `oncoanalyser` the local bundle directory must be provided with
+`--ref_data_hmf_data_path`.
+
+#### Customising other data
+
+The path or URI to the VIRUSBreakend database can also be explicitly set with `--ref_data_virusbreakenddb_path`. There
+are additional arguments to manually set various other reference data files, please review the parameters documentation
+for the complete list.
+
+#### Staging reference data
+
+Default reference data can be staged locally with `oncoanalyser` by providing a samplesheet for the desired analysis and
+setting the `--prepare_reference_only` argument. The samplesheet and `oncoanalyser` configuration will determine the
+relevant reference data to download. For example the following command will download the `GRCh38_hmf` genome plus
+indices, reference data, and databases required to run a WGTS analysis for tumor/normal DNA with tumor RNA:
+
+```csv title="samplesheet.csv"
+group_id,subject_id,sample_id,sample_type,sequence_type,filetype,filepath
+P1_wgts,P1,SA,normal,dna,bam,/path/to/P1.SA.normal.dna.wgs.bam
+P1_wgts,P1,SB,tumor,dna,bam,/path/to/P1.SB.tumor.dna.wgs.bam
+P1_wgts,P1,SC,tumor,rna,bam,/path/to/P1.SC.tumor.rna.wts.bam
+```
+
+```bash
+nextflow run nf-core/oncoanalyser \
+  -profile docker \
+  -revision 1.0.0 \
+  --mode wgts \
+  --genome GRCh38_hmf \
+  --prepare_reference_only \
+  --input samplesheet.csv \
+  --outdir prepare_reference/
+```
+
+Executing the above command will download and unpack default reference data without running any analysis, and once
+complete the prepared reference files can found in `./prepare_reference/reference_data/1.0.0/<datetimestamp>/`. It is
+recommended to remove the Nextflow work directory after staging data to free disk space.
+
+For `oncoanalyser` to use locally staged reference data a custom config can be used:
+
+```text title="refdata.local.config"
```

Comment:
```suggestion
```groovy title="refdata.local.config"
```

Reply:
- Change made in https://github.com/nf-core/oncoanalyser/pull/70